tests/test_size.py

```diff
@@ -1,12 +1,14 @@
 import ast
 
-from vulture.lines import estimate_lines
+from vulture.lines import count_lines
 
 
 def check_size(example, span):
```

Comment:
The second argument should be a single number, not a list:

    def check_size(code, size):
        module = ast.parse(code)
        for node in module.body:
            if isinstance(node, ast.ClassDef) and node.name == 'Foo':
                assert count_lines(node) == size
        assert False, 'failed to find top-level class "Foo" in code'